settings.gradle

```diff
@@ -1,44 +1,45 @@
 rootProject.name = "grpc"
-include ":grpc-core"
```

Comment:
Why were these lines changed?

Reply:
reverted to reduce review complexity. i'll issue other PR for sorting the order.